lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -143,10 +143,11 @@ sub list_ajax {
                 flavor  => $job->FLAVOR  // '',
                 arch    => $job->ARCH    // '',
                 build   => $job->BUILD   // '',
-                testtime => $job->t_finished . 'Z',
-                result   => $job->result,
-                group    => $job->group_id,
-                state    => $job->state
+                testtime      => $job->t_finished . 'Z',
+                result        => $job->result,
+                group         => $job->group_id,
+                comment_count => $job->comments->count,
```

Comment:
Not sure how often we already had this discussion, but $job->comments->count in a loop is horror. And /tests is slow enough - I don't know who has any interest in making it any more slower. 

So I will *not* merge this - as it adds little value but a lot of frustration to those waiting for /tests to finish loading.

Better make it server side pagination before you add more features to it.